src/index.js

```diff
@@ -110,7 +112,7 @@ const { updateSelfPeerRecord } = require('./record/utils')
  * @property {KeychainOptions & import('./keychain/index').KeychainOptions} [keychain]
  * @property {MetricsOptions & import('./metrics').MetricsOptions} [metrics]
  * @property {import('./peer-routing').PeerRoutingOptions} [peerRouting]
- * @property {PeerStoreOptions & import('./peer-store/persistent').PersistentPeerStoreOptions} [peerStore]
+ * @property {PeerStoreOptions & PersistentPeerStoreOptions} [peerStore]
```

Comment:
Why can't we use the types defined in https://github.com/libp2p/js-libp2p/blob/3bed7b4cb25157191b29d4b058cb6222eb3bfcd7/src/peer-store/persistent/index.js#L35 instead of duplicating them?

Reply:
I suppose that's an issue with `pbjs` when we use that along with `const PersistentPeerStore = require('./peer-store/persistent')` at the beginning, it'd generate `PersistentPeerStore.PersistentPeerStoreOptions` and cause the error 

```
@chainsafe/lodestar: ../../node_modules/libp2p/dist/src/index.d.ts:413:37 - error TS2503: Cannot find namespace 'PersistentPeerStore'.
@chainsafe/lodestar: 413     peerStore?: (PeerStoreOptions & PersistentPeerStore.PersistentPeerStoreOptions) | undefined;
```

this does not look fancy but I can't find other ways to get through that, any suggestions @vasco-santos ? Upgrading to the latest version of `pbjs` did not help

Reply:
ok, let's keep it. It is just a property